report_display_name_in_footer/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,20 @@
+This module extends the pagination functionality in reports to display the name of the
+document on each page of PDFs, allowing you to know which document each page belongs to
+and in what order they should appear.
+
+Pagination will appear on the left side and document name on the right side of the
+footer.
+
+### Implementation Details
```

Comment:
Can you move this section to ROADMAP.md?

Because we are doing a replace and maybe if anyone inherit the core reports, an error could raise.